source/index.ts

```diff
@@ -2,6 +2,7 @@
  * @file Automatically generated by barrelsby.
  */
 
+export * from "./conditionally";
```

Comment:
we wouldn't want to include `conditionally` in the index file, so we'd have to instruct `barrelsby` to ignore the file

Reply:
Oh, thanks for the tip, I'll take a look (and learn about "barrelsby")